### DISCUSSION TITLE
关于白名单的使用方法？

### DISCUSSION THREAD

**Discussion by FRANKASEE** [Guovin/iptv-api#749] [Q&A] [answered]
新的白名单使用方法是和原来的一样在每个URL后面添加$!即可实现（如：广东珠江,http://xxx.m3u$/! ），还是直接在白名单内添加URL地址就可以？

针对整个订阅源想都设置白名单模式是直接添加订阅地址(如:http://192.168.6.158:35456/txt/3），
还是逐行添加?
（如:CCTV1,http://192.168.6.158:35455/gaoma/cctv1.m3u8
CCTV1,http://192.168.6.158:35455/itv/6000000001000029752.m3u8?cdn=wasusyt
CCTV1-50-FPS,http://192.168.6.158:35455/itv/5000000004000002226.m3u8?cdn=bestzb
CCTV1-HEVC,http://192.168.6.158:35455/itv/1000000005000265001.m3u8?cdn=ystenlive
CCTV2-高码,http://192.168.6.158:35455/gaoma/cctv2.m3u8
CCTV2,http://192.168.6.158:35455/itv/1000000001000023315.m3u8?cdn=ystenlive
CCTV2-50-FPS,http://192.168.6.158:35455/itv/6000000001000014161.m3u8?cdn=wasusyt
CCTV2-HEVC,http://192.168.6.158:35455/itv/1000000001000023315.m3u8?cdn=ystenlive
CCTV2,http://192.168.6.158:35455/tptv/G_CCTV-2-HD-265-8M.m3u8）

另外还有一个问题，http://aktv.top/live.txt  订阅内的港澳台接口，如果添加到订阅内会被检测到盗源，如何能直接添加到最终生成的文件内，而且不被检测盗源?

**Comment by Guovin** [comment, marked as answer]
#750 重复
